examples/robot.cc

```diff
@@ -327,7 +327,7 @@ int main(int argc, char *argv[])
         paths.FindSharedLibrary(robotLib));
 
   std::unordered_set<std::string> drivePlugins =
-      loader.PluginsImplementing("ignition::plugin::examples::Drive");
+      loader.PluginsImplementing("Drive");
```

Comment:
These ones need to stay, they're strings

Reply:
Fixed! https://github.com/gazebosim/gz-plugin/compare/1c880de399144e1efd8f2a4afebc8dbea150b7c7..97fd2e82b0f72bb87f1129e9de8a76c5aaf7e2a3